examples/mnist_3d/Experiment.toml

```diff
@@ -0,0 +1,33 @@
+[experiment]
+project           = "CliMAgen.jl"
+name              = "mnist_3d"
+savedir           = "output"
+rngseed           = 42
+logging           = false
+nogpu             = false
+
+[data]
+batchsize         = 16
+
+[model]
+inchannels        = 1
```

Comment:
should we also support nx, ny, nt? or nspatial?

Reply:
Maybe also a good idea for the overhaul :).